prometheus/templates/pg_alert.rules.tmpl

```diff
@@ -0,0 +1,48 @@
+
+- name: Postgres Memory Available
+  rules:
+%{ for postgres_instance in alertable_postgres_instances ~}
+  - alert: Postgres memory available low (instance - ${postgres_instance})
+    expr: freeable_memory_bytes{service=~"${postgres_instance}"} < (512 * 1024 * 1024)
+    for: 5m
+    annotations:
+      summary:     ${postgres_instance} low memory available
+      dashboard:   ${postgres_dashboard_url}&var-Services=${postgres_instance}
+      description: "Postgres Memory available is less than 512M (current value: {{ $value }})"
+    labels:
+      severity:    high
+      app:         ${postgres_instance}
+      environment: production
```

Comment:
As discussed, let's not hardcode the labels

Reply:
removed severity and environment